services/QuillLMS/db/migrate/20230413140558_create_canvas_accounts.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateCanvasAccounts < ActiveRecord::Migration[6.1]
+  def change
+    create_table :canvas_accounts do |t|
+      t.string :external_id, null: false
+      t.references :user, foreign_key: true, null: false
+      t.references :canvas_config, foreign_key: true, null: false
```

Comment:
I think we should talk through this table. I'm guessing that this came up after we chatted [about this schema](https://www.notion.so/quill/Canvas-Configurations-33c5e500e9154a90b3bac1d963e2ef67), but I'm wondering if we are concerned that we are creating multiple paths to `canvas_config` from a `user` that can drift from each other, 
i.e. `user->canvas_config`, `user->school->canvas_config`.

We might consider linking to `school` here or not linking to anything and always going through the `user->school->canvas_config` path. Or keep what you have here and code around possible duplication.

Either way, I think it's worth talking through the pros and cons.

Reply:
While I was aware of the duplicate paths, my goal was to try to keep the schema structure similar to the `learn_worlds_account`.  After looking through this again, I would agree that the multiple paths should be removed.